lib/pf/profile/filter/connection_type.pm

```diff
@@ -0,0 +1,58 @@
+package pf::profile::filter::connection_type;
+=head1 NAME
+
+pf::profile::filter::connection_type - Class for connection_type
+
+=cut
+
+=head1 DESCRIPTION
+
+pf::profile::filter::connection_type
+
+=cut
+
+use strict;
+use warnings;
+use Moo;
+extends 'pf::profile::filter::key';
+
+=head1 ATTRIBUTES
+
+=head2 key
+
+Setting the key to connection_type
+
+=cut
+
+has '+key' => ( default => sub { 'last_connection_type' } );
+
+=head1 AUTHOR
+
+Inverse inc. <info@inverse.ca>
+
+Minor parts of this file may have been contributed. See CREDITS.
```

Comment:
I updated my copyright template. And fixed all new files that have this information.